packages/cli/src/commands/serve/playground.ts

```diff
@@ -69,6 +73,7 @@ export const playgroundMiddlewareFactory = ({
           renderGraphiQL({
             defaultQuery,
             endpoint: graphqlPath,
+            subscriptionsEndpoint: `ws://${hostname}:${port}${graphqlPath}`,
```

Comment:
We don't want to use WebSockets as default transport in GraphiQL. So we prefer to keep SSE by default.

Reply:
Ok, no issues with that. Thing is that right now GrpahiQL seems to have some issue with the current configuration as its not possible to run a subscription. Is this something that needs to be fixed in GraphiQL?

Reply:
It might very well be that we misconfigured something, but we tried a lot of different things and couldn't get it to work.

Reply:
It seems something has been changed in graphql-helix. Reverting it back to `1.2.3` fixes that.

Reply:
@ardatan I see. Good to know it wasn't us :D

In case you want to keep the latest graphql-helix version you can have a look at commit
https://github.com/ntziolis/graphql-mesh/commit/6c0b387bed772d560434fa741b56b14159b173e7

Just tested and graphiql works just fine with this approach while not using ws as protocol.

Reply:
@ntziolis If someone accesses GraphiQL from some other hostname, this would fail as well for example on CodeSandbox subscriptions won't work in this case because it will always look for `localhost`. I created a PR to revert this behavior on GraphQL Helix instead.
https://github.com/contrawork/graphql-helix/pull/32

Reply:
Ah I see, even better.